.husky/pre-push

```diff
@@ -2,3 +2,5 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npm run test
+npm run make-badges
+git add README.md
```

Comment:
Please add a new line character at the end of the file

Reply:
fixed